### PR TITLE
Supposed to be good. Now with machine names

### PR DESCRIPTION
**Pull request by SetMyDream**

### CODE REVIEW

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    protected String machineName = "Bulldozer";
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Bulldozer.java:4]:
```suggestion
    private String machineName = "Bulldozer";
```

File: src/main/java/core/basesyntax/Bulldozer.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        System.out.print(machineName + " is working!");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.print(machineName + " just stopped to work!");
+    }
+
+    @Override
+    public void getName(String name) {
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/Bulldozer.java:17]:
```suggestion
    public void setName(String name) {
```

File: src/main/java/core/basesyntax/MainApp.java [hunk 3]
```diff
@@ -12,5 +12,17 @@
...
+        Machine excavator = new Excavator();
+        Machine bulldozer = new Bulldozer();
+        Machine track = new Track();
+        Machine namedTrack = new Track();
+        namedTrack.getName("F1 track");
+        Machine[] park = {excavator, bulldozer, track, namedTrack};
+        for (Machine machine : park) {
+            machine.doWork();
+            machine.stopWork();
+            //Надеюсь, не принципиально, что бы все работали одновременно.
```
Comment (Sofasmile) [inline src/main/java/core/basesyntax/MainApp.java:25]: no)
let's remove this comment)

Comment (SetMyDream) [reply src/main/java/core/basesyntax/MainApp.java]: Done!

**Pull request closed without merging**